### ISSUE TITLE
First Stage in Desperate Need of Re-Balance + Oversight in Modelling (Missed / didn't model about 2/3rds of the 1st stage tank)

### ISSUE THREAD

**Issue by Raptor22a** [cmdrads/ImprovedSeaDragon#2]
The fuel level of the 1st stage, and the consumption of the 1st stage engine, are wildly unbalanced; when there's no payload on it, it only gives a DeltaV of 364m/s... and a burn time of _**THREE SECONDS**_. It also has an insane TWR for the first stage, with a TWR at launch of a payload-less vehicle being 12.27. While I know that it needs a high TWR in order to loft heavy payloads, a burn time of 3 seconds just is not congruent with reality; it'd barely have gotten out of the water by the time the 1st stage was expended. I've tried having the throttle reduced to reduce fuel consumption, but I can barely get it to have a burn of over 30 seconds while being able to have a TWR of more than 1. If I have the thrust limiter set to 8.5%, it will get a TWR of 1.04 and a burn time of only 33 seconds. 

I haven't been able to get more than 2.5km/s of DeltaV when launching from sea level

Also, I noticed that the 1st stage tank doesn't get anywhere near up to the 2nd stage engine bell, and there's an inexplicably unused node below the engine bell but above the attach node for the tank. I'm able to fit two whole tanks inside the expandable nozzle section; however, due to the insane fuel consumption and massively insufficient amount of fuel in the tank, reducing the TWR to 11.5% to try to get the most burn time out of the engine (TWR of 1.03) results in only 49 seconds of burn time and 535 m/s of DeltaV. In most of the designs I've seen for Sea Dragon, the 1st stage tank is designed to stick up inside the engine bell of the 2nd stage, maximizing the volume available; leaving enough space to fit an entire 2nd tank in there just _empty_ seems highly strange. 
### But, I believe I've discovered your problem: You mistakenly only modeled the kerosene tank for the 1st stage.
Please refer to this image here:
http://www.astronautix.com/graphics/s/sdragon.jpg
Or this here:
https://upload.wikimedia.org/wikipedia/commons/6/6a/SeaDragonRocketv2.gif

The Kerosene tank (denoted in Orange) is what you currently have modeled there, with the rounded top.
But, if you look at the diagram from Astronautix there, you'll see that the LOX tank (Denoted in Blue) has a cone-shaped top that extends into the nozzle of the 2nd stage engine and right up into the throat of it. It's less apparent in the 2nd diagram, but if you look, you can see that the conical top of the LOX tank is separate from the curved parabolic shape of the 2nd stage engine. If you zoom in, you'll also see how the conical top of the LOX tank turns into a cylinder up at the very top, where the 1st stage LOX vert. vent is denoted, along with the stage pressurization unit. All in all, the LOX tank extends _aaall_ the way to Station 279.14, well up into the throat of the 2nd stage engine and practically inside its combustion chamber.

**So, if you were calculating the propellant purely based on the volume of the part there, then the reason for all of this is evident: it's trying to calculate both liquid fuel and oxidizer fitting only inside the oxidizer tank, which is only about 1/3 of the 1st stage** Of course, the fuel consumption rate would still be incredibly high even with 3 times the volume, but if you changed the consumption rate and fixed the fuel tanks to be the correct size, I think it'd all work swimmingly.

Another bit of an oddity is that, according to the original blueprints (shown in the wikimedia link), there was a small CH4 tank for the 1st stage. I'm not sure how much CH4 it's supposed to hold or why it had CH4 in addition to RP-1 - if you want to add that detail, you can, but I'd wager making a tri-propellant engine in KSP is more hassle than it's worth, so only consider adding it if you want to challenge yourself and/or be as accurate as possible.

At some point, it'd be nice to see if you added in the recovery flare used for decelerating the 1st stage for splashdown. 

Finally, for the TVC engines, I think that the fairings that you made for them need to have their attach point recessed inward. The J-2s that you recommended in the previous post seem to fit in there well, but pretty much all of the designs of Sea Dragon have those TVC engines almost entirely covered by the fairings, with only a little bit of the nozzle poking out. In the original blueprints from the Wikimedia link, they don't even poke out of the fairing, being completely covered by it.

In its current state, the mod is almost entirely unusable due to the balance issues. Not to mention the fact that the ballast is missing (I saw that you closed that issue - I assume that's indicating you've fixed it. Any ETA on when you can push commit?), and the lack of gimbal control for the 1st stage engine and center of lift being FAR above the center of mass at launch makes it extremely prone to flipping over.

### I'm not trying to be overly-critical, mind you. I think that the mod looks excellent, and the visuals are great. But, there's a lot of work that needs to be done to get the functionality down-pat; I'm confident that you'll be able to do it

**Comment by cmdrads** [comment]
ima be fully honest with you chief, I already know all of these issues, and I have no intent on fixing them any time soon because I'm busy with uni and have better things to do.

Dont use the mod if you don't find it usable, simple as really.

I know it's unbalanced, I know shits modelled wrong, I genuinely do not have the time or energy to finish it, there's a reason why I haven't updated it in months, I have uni and real life commitments to care about.

**Comment by cmdrads** [comment]
before you or anyone else asks if they can take over the mod.

No, lol.

I'll get around to fixing everything one day.

that day isn't today, or soon, it'll be fixed when I have time

**Comment by cmdrads** [comment]
since you have no immediate fixable issue, and instead gave me an essay of things I know already, I will consider this "issue" solved and shall close the thread

**Comment by Raptor22a** [comment]
> before you or anyone else asks if they can take over the mod.
> 
> No, lol.
> 
> I'll get around to fixing everything one day.
> 
> that day isn't today, or soon, it'll be fixed when I have time

So, you won't be fixing it any time soon, yet won't let anyone else take over to fix it for you? Just let it sit and languish, barely functional?

Frankly, putting it as "closed" when you haven't done anything to resolve it seems hasty and premature. Not to mention that yes, it is an issue, as the mod is almost completely unusable right now, so considering it "solved" when you have done literally nothing to fix it is just... wrong.

Trust me, I get what it's like to have university classes - I'm in the middle of final exams for my sophomore year of my aerospace engineering undergraduate. But, if you don't have time to fix it right now, just leave it as open so that it's an indicator for you to come back to it later. Marking it as closed just seems disingenuous and as if you're sweeping multiple issues that make your mod completely broken under the rug.

**Comment by cmdrads** [comment]
you know, posting passive aggressive essays is not doing wonders for my motivation.

I am marking these as closed and "unplanned" because that's the reality.

Its unplanned

**Comment by QuantifyGG** [comment]
Raptor, it's important to remember that Aaron doesn't actually owe you anything, being a bit more courteous w.r.t. his time might be a good idea.

**Comment by Raptor22a** [comment]
> Raptor, it's important to remember that Aaron doesn't actually owe you anything, being a bit more courteous w.r.t. his time might be a good idea.

**I never said that he owed anyone anything.** Don't put words that I didn't say into my mouth. I literally acknowledged that I know what it's like to be extremely busy in university. There is, however, a difference between saying something along the lines of "I'm already aware of this, but thank you for bringing it to my attention. I'm busy with college right now, but I'll eventually get to it later" and saying what effectively boils down to "You wasted your time telling me this, and I'm not going to do any work on fixing any of this, not planning to either".

If he had said something along the lines of the former, I would have said "great! Good luck with your courses" and went along my way. But, the fact that he was so extremely dismissive of the feedback that I put some serious time and effort into in order to try my best to be helpful and give him some assistance seriously struck a nerve. It's sticking out your hand to help and having it slapped away.

**Comment by Raptor22a** [comment]
> Raptor, it's important to remember that Aaron doesn't actually owe you anything, being a bit more courteous w.r.t. his time might be a good idea.

**I never said that he owed anyone anything.** **_Don't put words that I didn't say into my mouth._** I literally acknowledged that I know what it's like to be extremely busy in university. There is, however, a difference between saying something along the lines of "I'm already aware of this, but thank you for bringing it to my attention. I'm busy with college right now, but I'll eventually get to it later" and saying what effectively boils down to "You wasted your time telling me this, and I'm not going to do any work on fixing any of this, not planning to either".

If he had said something along the lines of the former, I would have said "great! Good luck with your courses" and went along my way. But, the fact that he was so extremely dismissive of the feedback that I put some serious time and effort into in order to try my best to be helpful and give him some assistance seriously struck a nerve. It's sticking out your hand to help and having it slapped away.

I don't see how anything I did was discourteous. While I pointed out that the mod is broken, I said that I'm confident in the mod author's skills to be able to fix them, offering words of encouragement and suggestions to help them get started. They are the ones who came in here and essentially said "lol, no, go away."

 
Add heading textAdd bold text, <Ctrl+b>Add italic text, <Ctrl+i>
Add a quote, <Ctrl+Shift+.>Add code, <Ctrl+e>Add a link, <Ctrl+k>
Add a bulleted list, <Ctrl+Shift+8>Add a numbered list, <Ctrl+Shift+7>Add a task list, <Ctrl+Shift+l>
Directly mention a user or team
Reference an issue, pull request, or discussion
Add saved reply
Slash commands
Leave a comment
No file chosen
Attach files by dragging & dropping, selecting or pasting them.
Styling with Markdown is supported
Remember, contributions to this repository should follow our [GitHub Community Guidelines](https://docs.github.com/articles/github-community-guidelines).
Assignees
No one assigned
Labels
None yet
Projects
None yet
Milestone
No milestone
Development
No branches or pull requests

Notifications
Customize
You’re receiving notifications because you authored the thread.
3 participants

**Comment by Raptor22a** [comment]
> Raptor, it's important to remember that Aaron doesn't actually owe you anything, being a bit more courteous w.r.t. his time might be a good idea.

**I never said that he owed anyone anything.** _**Don't put words that I didn't say into my mouth.**_ I literally acknowledged that I know what it's like to be extremely busy in university. There is, however, a difference between saying something along the lines of "I'm already aware of this, but thank you for bringing it to my attention. I'm busy with college right now, but I'll eventually get to it later" and saying what effectively boils down to "You wasted your time telling me this, and I'm not going to do any work on fixing any of this, not planning to either".

If he had said something along the lines of the former, I would have said "great! Good luck with your courses" and went along my way. But, the fact that he was so extremely dismissive of the feedback that I put some serious time and effort into in order to try my best to be helpful and give him some assistance seriously struck a nerve. It's sticking out your hand to help and having it slapped away.

I don't see how anything I did was discourteous. While I pointed out that the mod is broken, I said that I'm confident in the mod author's skills to be able to fix them, offering words of encouragement and suggestions to help them get started. They are the ones who came in here and essentially said "lol, no, go away."
